3_summarize/src/calculate_max_benthic_area.R

```diff
@@ -0,0 +1,17 @@
+
+calculate_max_benthic_area <- function(target_name, all_hypsos) {
```

Comment:
I think calling this variable, function, and file as "max" is a bit confusing. What we are calculating is the lake's benthic area. In the other TOHA functions, we are calculating the portion of the benthic area that meets certain criteria. 

I'd suggest calling it `benthic_area` or `total_benthic_area` if we need to make it clear this is for the whole lake.

Reply:
Sure, can do. Was just using the terminology that we've been using in our conversations and emails. Agree that total better reflects what we are talking about.